KsApi/models/Project.swift

```diff
@@ -96,6 +97,11 @@ public struct Project {
       return floor(Float(self.pledged) * self.staticUsdRate)
     }
 
+    /// Total amount currently pledged to the project converted to USD irrespective of the user's selected currency
```

Comment:
/// Total amount currently pledged to the project, converted to USD, irrespective of the users selected currency